docs/source/reservations/existing-machines.rst

```diff
@@ -92,6 +92,10 @@ Equivalently, use ``resources.infra: ssh/<node_pool_name>`` in a task YAML:
 
 See more customization options and details about SSH Node Pools in the rest of this guide.
 
+.. tip::
+
+    When working with VMs with GPUs, the VMs must have `nvidia-smi` installed.
```

Comment:
```suggestion
    When working with GPU instances, GPU drivers must be installed on the host. Verify by running ``nvidia-smi``.
```

Reply:
Should we move this to prerequisites? https://docs.skypilot.co/en/latest/reservations/existing-machines.html#details-prerequisites